zeppelin-server/src/main/java/org/apache/zeppelin/rest/AdminRestApi.java

```diff
@@ -81,4 +90,51 @@ public AdminRestApi(AdminService adminService) {
 
     return Lists.newArrayList(adminService.getLogger(loggerRequest.getName()));
   }
+
+  /**
+   * Change quartz thread pool size REST API.
+   *
+   * @param message - JSON with poolSize value.
+   * @return JSON with status.OK
+   */
+  @POST
+  @Path("cron/pool/{id}/poolSize")
```

Comment:
Решили же просто 
```suggestion
  @Path("cron/pool/{id}")
```
Так как `poolSize` будет полем внутри запроса.